app/controllers/users.js

```diff
@@ -3,11 +3,18 @@ const { createUser, encryptPassword } = require('../services/users');
 
 exports.signUp = ({ body }, res, next) => {
   logger.info('encrypting password');
-  encryptPassword(body.password)
+  return encryptPassword(body.password)
     .then(encryptedPassword => createUser({ ...body, password: encryptedPassword }))
     .then(user => {
+      const userFormated = {
```

Comment:
Generally, we use `serializers`. Its responsibility is to format the data for the response. Remember, the response always must be in snake_case, not in camelCase